server/controllers/lecture-controller.js

```diff
@@ -357,11 +358,11 @@ const sendEmailsForCancelledLecture = async (lectureId) => {
   const emailBody = (
     name,
     surname,
-    lectureName,
+    //lectureName,
```

Comment:
Usually it's seen as a bad practice to commit files containing commented lines of code.